.packit.yaml

```diff
@@ -47,8 +47,8 @@ jobs:
     targets:
       - centos-stream+epel-next-8-x86_64
       - centos-stream+epel-next-8-aarch64
-      - centos-stream+epel-next-9-x86_64
-      - centos-stream+epel-next-9-aarch64
+      - epel-9-x86_64
+      - epel-9-aarch64
```

Comment:
should the epel-next-8 above also changed?

Reply:
We will be removing all el8 stuff once centos8 stream goes EOL end of next month, so let's leave it as is for now.